### ISSUE TITLE
Question regarding epoch vs. accuracy

### ISSUE THREAD

**Issue by sigmoidx** [kuangliu/pytorch-cifar#41]
Hi,

I have a simple question regarding the epoch vs. accuracy. 

Like these, 
VGG16 | 92.64%
ResNet18 | 93.02%
ResNet50 | 93.62%

If someone made his own network and trained for 100,000 epochs and it seems to show 95% accuracy at epoch 100,000 at max. and the accuracy declines after that. 
And what if ResNet18 showed 93.02% max. at epoch 200 and accuracy declines after that, 
Can we say his custom network is better than ResNet18? 

Do we have to consider the same number of epoch to compare accuracy between two networks? or max. accuracy only (not considering number of epoch.)?

Let's say at 200 epoch, if ResNet18 has 93.02% and his custom network has around 80%, is it possible to say ResNet18 is better?

Thanks,
